examples/flutter_gallery/lib/demo/cupertino/cupertino_switch_demo.dart

```diff
@@ -62,7 +62,7 @@ class _CupertinoSwitchDemoState extends State<CupertinoSwitchDemo> {
                         onChanged: null,
                       ),
                       Text(
-                        'Disabled'
+                        'Enabled'
```

Comment:
Understood, if you'd like to help us clean this up, let's name the first one 'Enabled - On', this one 'Disabled - On' and the next one 'Disabled - Off'.